examples/android/androidGuiExample/Makefile

```diff
@@ -0,0 +1,13 @@
+# Attempt to load a config.make file.
```

Comment:
Do we want to add these project files?

Reply:
I think that these should not be there as no example carries a makefile. I'll remove these